src/test/java/RegistrationTest.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+import pages.RegistrationPage;
+
+import static org.testng.Assert.assertTrue;
+
+public class RegistrationTest {
+    WebDriver driver;
+    RegistrationPage registrationPage;
+
+    @BeforeClass
+    public void setup() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://percab.uzairways.com/auth/register");
+        registrationPage = new RegistrationPage(driver);
+    }
+
+    @Test
+    public void testUserRegistration() throws InterruptedException {
+        registrationPage.enterEmail("testuser@example.com");
+        registrationPage.enterFirstName("Aziza");
+        registrationPage.enterLastName("Makhmudova");
```

Comment:
Please create variables for test data. It's better to not hardcode the value in the test. 

if any value changes, if it's hardcoded  - we will need to update it in every place it's mentioned, if we have a constant variable for that - we will need to update it once in the variable